numpy/distutils/mingw32ccompiler.py

```diff
@@ -390,6 +391,13 @@ def _build_import_library_amd64():
                   (out_file))
         return
 
+    # didn't exist in virtualenv, maybe in base distribution?
+    base_file = os.path.join(sys.base_prefix, 'libs', out_name)
+    if os.path.isfile(base_file):
+        log.debug('Skip building import library: "%s" exists' %
+                  (base_file))
```

Comment:
Agreed, and upon inspection, string formatting should not be done in calls to log.debug in the first place. I'll submit a revised version. Some projects don't like pull requests to be rebased while in review, though; what is your stance on that?